src/core/core/command_manager.cpp

```diff
@@ -418,5 +417,29 @@ namespace dsn {
             return "repeat command completed";
         }
         );
+
```

Comment:
for rcli, the 'help' command will not be passed to remote, but just get local help info.